bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/models/form/FieldType.java

```diff
@@ -35,7 +35,9 @@ public enum FieldType {
     PANEL("panel"),
     FORM("form"),
     CHECKBOX_GROUP("checkbox-group"),
-    IMAGE("image");
+    IMAGE("image"),
+
+    IMAGECHOICE("imagechoice");
```

Comment:
fieldType 'imagechoice' does not exists, I believe it should be 'checkbox'

Reply:
The reason for creating a new field type is that the image choice component is a composite multi/single selection type, and it will not store values as an enum.